### PR TITLE
airgap-upgrade-doc

### PR DESCRIPTION
**Pull request by ddjjia**
*Issue #, if available:*
[improve air gapped upgrade doc#1503](https://github.com/aws/eks-anywhere-internal/issues/1503)

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by chrisnegus — COMMENTED** [review]
I agree with @d8660091 that this content should go into the upgrade section. If you decide to move the content there, I would suggest having a pointer to that content from this document.

File: docs/content/en/docs/clustermgmt/cluster-upgrades/airgapped-upgrades.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Upgrade airgapped cluster"
+linkTitle: "Upgrade airgapped cluster"
+weight: 20
+aliases:
+   /docs/tasks/cluster/cluster-upgrades/airgapped-upgrades/
+description: >
+  How to perform eks-anywhere upgrade for an airgapped cluster
+---
+If you want to upgrade EKS Anywhere version, or your airgapped cluster upgrade requires EKS Anywhere version upgrade, perform the following:
```
Comment (d8660091) [inline docs/content/en/docs/clustermgmt/cluster-upgrades/airgapped-upgrades.md:10]:
```suggestion
If you want to upgrade EKS Anywhere version, or your airgapped cluster upgrade requires EKS Anywhere version upgrade in airgapped environment, perform the following steps to prepare new artifacts in your registry mirror:
```

**Review by chrisnegus — COMMENTED** [review]
Nice.
/lgtm

**Pull request merged**